test/http.Spec.js

```diff
@@ -0,0 +1,68 @@
+"use strict";
+
+var chai = require('chai');
+var chaiHttp = require('chai-http');
+var expect = require('chai').expect;
+
+chai.use(chaiHttp);
+
+describe('HTTP servers', function() {
+
+  it('should respond to request to /time with status code 200', function(done) {
+
+    chai.request('localhost:3000')
+          .get('/time')
+          .end(function (err, res) {
+            expect(err).to.be.null;
+            expect(res).to.have.status(200);
+            done();
+          });
+
+  });
+
+  it('server time in seconds should equal test time in seconds', function(done) {
+    var time = new Date().getTime();
+    chai.request('localhost:3000')
+          .get('/time')
+          .end(function (err, res) {
+            expect(res.text).to.equal(Math.floor(time/1000).toString());
+            done();
+          });
+
+  });
+
+  it('server should send a greeting to the name in the GET request', function(done) {
+
```

Comment:
@dxeware since there is only one statement in this "It" block, no need to add additional line breaks